server/test/routes/article.test.js

```diff
@@ -0,0 +1,372 @@
+/* eslint-disable */
+import chai from 'chai'
+import chaiHttp from 'chai-http'
+import server from '../../index';
+import tokenService from '../../utils/services/tokenService';
+import mockData from './mockData';
+
+const testUser = mockData.user7; 
+
+const { expect } = chai;
+const baseUrl = '/api/articles/';
+const SIGN_UP_ROUTE = '/api/users';
+chai.use(chaiHttp);
+let slug;
+
+describe('Testing articles routes', () => {
+    let token = tokenService.generateToken(testUser, '3d');
+      describe('Creating an Article', () => {
+        it('Should create an Article sucessfully with title, description and/or body specified', (done) => {
+            chai.request(server)
+            .post(`${baseUrl}`)
+            .send({
+                article:{
+                    title: 'A new title',
+                    description: 'A new description',
+                    body: 'A new body'
+                }
+            })
+            .set('Authorization', token)
+            .end((err, res) => {
+                expect(res.status).to.equal(201);
+                expect(res.body).to.have.property('article');
+                slug = res.body.article.slug;
+                done();
+              });
+        })
+      })
+      describe('Creating an Article with invalid parameters', () => {
+        it('Should return a 400(Bad request) error', (done) => {
+            chai.request(server)
+            .post(`${baseUrl}`)
+            .send({
+                article:{
+                    title: 1234,
+                    description: 1234,
+                    body: 12234
+                }
```

Comment:
Your work is looking good, but you will need to check the indentation of your test file. We are using two space indentation. You can do well to update the indentation